homeassistant/components/logbook.py

```diff
@@ -53,8 +68,19 @@ def log_entry(hass, name, message, domain=None, entity_id=None):
 
 def setup(hass, config):
     """ Listens for download events to download files. """
-    hass.http.register_path('GET', URL_LOGBOOK, _handle_get_logbook)
+    # create service handler
+    def notify_message(notify_service, call):
```

Comment:
This seems like a very complex way of defining a service. Consider doing something similar to [this service](https://github.com/balloob/home-assistant/blob/dev/homeassistant/components/thermostat/__init__.py#L143-L166)